src/azure-cli/azure/cli/command_modules/hdinsight/custom.py

```diff
@@ -490,3 +526,201 @@ def execute_hdi_script_action(cmd, client, resource_group_name, cluster_name,
     ]
 
     return client.execute_script_actions(resource_group_name, cluster_name, persist_on_success, script_actions_params)
+
+
+# pylint: disable=redefined-builtin
+def create_autoscale(cmd, client, resource_group_name, cluster_name, type, min_count=None,
+                     max_count=None, timezone=None, days=None, time=None, count=None):
```

Comment:
>count [](start = 74, length = 5)

Do you think if `--worknode-count` is a better name? Considering we also have edgenode concept in HDI

Reply:
should we change min-count to min-workernode-count too? My opinion is that we only support autoscale workernode, right? so if it is autocale, the count means workernode count. I am not sure. 
 I remembered Shangwei and you thinked min-workernode-count is redundantat our review. But Deepali said we should add workernode in last meeting, right?

---
In reply to: [466348597](https://github.com/Azure/azure-cli/pull/14692#discussion_r466348597) [](ancestors = 466348597)

Reply:
Yes, Deepali mentioned that we should add workernode, that's why I call up this topic. Both names look good to me.

Reply:
Changed --count to --workernode-count and  --min-count to --min-wokernode-count, --max-count to --max-workernode-count.

---
In reply to: [466822096](https://github.com/Azure/azure-cli/pull/14692#discussion_r466822096) [](ancestors = 466822096)